### DISCUSSION TITLE
WebGPU is available?

### DISCUSSION THREAD

**Discussion by deeivihh** [zen-browser/desktop#3926] [Q&A]
If I remember correctly, this feature is enabled by default in the [nightly versions of Firefox,](https://www.mozilla.org/es-ES/firefox/channel/desktop/#nightly) I've adjusted the settings in `about:config`, but it doesn't seem to be available for Zen, is it possible to enable it? If it is not available, would it be feasible to consider it as a feature to be added in the future?

**Comment by mr-cheffy** [comment]
`dom.webgpu.enabled` to true in about:config

**Reply by deeivihh** [reply]
I have activated all the options containing the word **webgpu** and according to [WebGPU Report](https://webgpureport.org/), it is not supported.

![options](https://github.com/user-attachments/assets/f27ebbea-5290-45e5-8315-1324b6d7cfbf)
![webgpu report](https://github.com/user-attachments/assets/e598e88d-f27c-4dad-83b6-dea645730276)

**Reply by different55** [reply]
How's it look on Firefox on the same machine?
